src/styles/index.scss

```diff
@@ -1,3 +1,116 @@
+$main-color: #060b35;
+
+$secondary-color: #616070;
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.product-card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &__image {
+    margin-bottom: 36px;
+    padding: 0 3px;
+    img {
+      width: 100%;
+      height: 134px;
+    }
+  }
+  &__title {
+    font-size: 12px;
+    line-height: 1.5;
+    margin: 0 0 4px;
+    font-weight: 500;
+    color: $main-color;
+  }
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 1.4;
+    color: $secondary-color;
+    margin: 0 0 16px;
+  }
+  &__stars {
+    margin-bottom: 24px;
+  }
+  &__price {
+    margin-bottom: 16px;
+  }
+}
+
+.product-stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url("./../images/star.svg");
+    &:not(:last-child) {
+      margin-right: 4px;
+      background-image: url("./../images/star-active.svg");
```

Comment:
it's better to handle all possible variants (stars--1, etc) - there should be reusable styles from the previous (`stars`) task